packages/ui/package.json

```diff
@@ -22,8 +20,27 @@
   "sideEffects": [
     "*.css"
   ],
+  "exports": {
+    ".": {
+      "default": "./src/index.ts"
+    },
+    "./definitions": {
+      "default": "./src/definitions.ts"
+    },
+    "./package.json": "./package.json"
+  },
```

Comment:
Not entirely sure if this is correct. What I wanted to achieve was a separate export of the component definitions, so they could be imported in `docs-ui` without running into Next.js SSR issues:

<img width="936" height="1039" alt="image" src="https://github.com/user-attachments/assets/1ef49de9-71f1-47fa-902c-384f92f7c731" />